void main(List<String> args) async {
  await Future.error("error");
}
